### ISSUE TITLE
Submenu (.usa-sidenav) links fail contrast minimum on hover

### ISSUE THREAD

**Issue by iamjolly** [18F/18f.gsa.gov#3238]
## Expected Behavior
All text on site will meet [WCAG 1.4.3 Contrast Minimum](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

## Actual Behavior
Page submenu links (`.usa-sidenav`) have extremely poor contrast (1.62:1) on `:hover`. The minimum contrast ratio must be 4.5:1. 

Screenshot: 
<img width="902" alt="screenshot from 18F about page showing low contrast link text on hover" src="https://user-images.githubusercontent.com/1093423/92929570-fc191880-f3fd-11ea-8248-734df1a4ad15.png">

## Steps to reproduce the behavior
- [ ] Open https://18f.gsa.gov/about/
- [ ] Hover over sidenav links

## This issue is done when:
- [ ] Contrast of link text meets or exceeds 4.5:1
